AnkiDroid/src/main/java/com/ichi2/anki/DeckPicker.kt

```diff
@@ -122,6 +123,7 @@ import java.io.File
 import kotlin.math.abs
 import kotlin.math.roundToLong
 
+val POSTPONE_MIGRATION_KEY = "secondWhenMigrationWasPostponedLast"
```

Comment:
extract migration interval to constant, I propose being more pushy: 5 days not 10

```suggestion
val POSTPONE_MIGRATION_KEY = "secondWhenMigrationWasPostponedLast"
val POSTPONE_MIGRATION_INTERVAL_DAYS = 5
```